src/tic_one.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _tic_one{T<:AbstractFloat}(zmin::T, pixx::T, incr::T, ra::Bool)
+    if ra
+        mul = T(4)
+    else
+        mul = T(60)
+    end
+    min1 = zmin*mul
+    incra = abs(incr)
+    rem = min1 % incra
+    sign = min1*incr
+
+    if sign > 0
+        tic1 = pixx - abs(rem)*pixx/incra
+        min2 = (min1 + incr -rem)/mul
+    else
+        tic1 = abs(rem)*pixx/incra
+        min2 = (min1 - rem)/mul
+    end
+    return min2, tic1
+end
+
+"""
+    tic_one(zmin, pixx, incr[, ra=true]) -> min2, tic1
+
+### Purpose ###
+
+Determine the position of the first tic mark for astronomical images.
+
+### Explanation ###
+
+For use in labelling images with right ascension and declination axes.
+This routine determines the position in pixels of the first tic.
+
+### Arguments ###
+
+* `zmin`: astronomical coordinate value at axis zero point (degrees
+   or hours).
+* `pixx`: distance in pixels between tic marks (usually obtained from TICS).
```

Comment:
When you refer to another function in AstroLib, please use this syntax: `[`tics`](@ref)`, so that there will be a clickable link in the HTML documentation at https://juliaastro.github.io/AstroLib.jl/latest/ref.html  I didn't use this syntax before because this feature is available only since we use Documenter.jl for the documentation,.